lease/lessor.go

```diff
@@ -220,6 +227,7 @@ func (le *lessor) Revoke(id LeaseID) error {
 		le.mu.Unlock()
 		return ErrLeaseNotFound
 	}
+	close(l.revokec)
```

Comment:
Should this close be in a `defer` since the deletion isn't totally complete yet? Not sure if it makes a difference but might as well play it safe...